it/v2/endpoints/RetrieveUkPropertyPeriodSummaryControllerISpec.scala

```diff
@@ -172,13 +173,11 @@ class RetrieveUkPropertyPeriodSummaryControllerISpec extends V2IntegrationBaseSp
 
```

Comment:
(Can't comment directly on it but) can we update `ifsResponseBody` on `line 115` to be `downstreamResponseBody`